cobbler option fix

keichwa opened the pull request:
starting with https://github.com/SUSE/spacewalk/issues/7964
close https://github.com/SUSE/spacewalk/issues/7964

keichwa commented:
@SchoolGuy 
Within comment markers, I added your list about "Background Information about the Synchronization Process".  I am uncertain whether it makes sense to publish it here.  I'd appreciate if you'd review the complete chapter as part of the "content review process" that Lana currently drives.

Loquacity commented on modules/client-configuration/pages/cobbler.adoc:
```diff
@@ -206,6 +207,27 @@ Doing so can cause errors.
+10. Run Post-Sync-Triggers (this can also be any number of shell scripts to my knowledge)
+
+////
+
```
I would classify this as a concept. For that reason alone, I would recommend rewording this into prose, and put it at the top of the file.

keichwa replied:
ok, done.

Loquacity commented:
> @SchoolGuy
> Within comment markers, I added your list about "Background Information about the Synchronization Process". I am uncertain whether it makes sense to publish it here. I'd appreciate if you'd review the complete chapter as part of the "content review process" that Lana currently drives.

Technical review, actually. Please put your name against the review here: https://github.com/SUSE/spacewalk/issues/9293

SchoolGuy commented:
@keichwa I looked at the complete page (not only your changes) and I guess that no one ever told me what actually is supported with cobbler.

Also a small sidenote we don't use anymore atftp we use tftp. (Looked it up on the 40 reference server in provo).

Running spacewalk-service after cobbler check does not make much sense because cobbler check requires the cobbler daemon to be already running.I don't know exactly what spacewalk-service   does but to my knowledge it is just a wrapper with some extras for starting the cobbler service...

There is also a link to fedorahosted wiki. This is a dead link as fedorahosted does not exist anymore. Upstream documentation is under review from @brejoc and will be backported from master to all version as soon as he is finished. Since there is not much change you could temporarily take this link instead: https://cobbler.github.io/manuals/2.8.0/3/5_-_Kickstart_Templating.html

All future documentation for cobbler will be found under https://cobbler.readthedocs.io/ and longer or more complicated guide under https://cobbler.github.io the wiki of Github found under https://github.com/cobbler/cobbler/wiki will contain only development specific information. The client documentation will be found under https://koan.readthedocs.io but this is also currently a work in progress! But until we reach this point of alignment there is a long road to go!

keichwa commented:
"Enno G." <notifications@github.com> writes:

> @keichwa I looked at the complete page (not only your changes) and I
> guess that no one ever told me what actually is supported with cobbler.

Yes, and there are still some steps to go.  Major chunks are still
commented out, block-wise with the //// markers like this:

////
invisible block
////

These blocks usually come with a small comment such as:

////
Commented out as per https://bugzilla.suse.com/show_bug.cgi?id=1136611 LKB 2019-05-29
....
////

Can you see this?  If yes, we'd highly appreciate if you'd take a closer
look and either simply enable it (if it is supported/working) or add a
comment what has to be changed.

If wanted you can create a PR directly in github or just add comments as
this one here :-)

> Also a small sidenote we don't use anymore atftp we use tftp. (Looked it
> up on the 40 reference server in provo).

Ok, I changed it accordingly.

> Running spacewalk-service after cobbler check does not make much sense
> because cobbler check requires the cobbler daemon to be already
> running.I don't know exactly what spacewalk-service does but to my
> knowledge it is just a wrapper with some extras for starting the cobbler
> service...

Thanks for spotting, I changed it.

> There is also a link to fedorahosted wiki. This is a dead link as
> fedorahosted does not exist anymore. Upstream documentation is under
> review from @brejoc and will be backported from master to all version as
> soon as he is finished. Since there is not much change you could
> temporarily take this link instead:
> https://cobbler.github.io/manuals/2.8.0/3/5_-_Kickstart_Templating.html

Ok, updated.  Thanks for the additional background info!

-- 
Karl Eichwalder            SUSE Software Solutions Germany GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
Geschäftsführer: Felix Imendörffer (HRB 247165, AG München)
